README.md

```diff
@@ -84,6 +84,7 @@ and a transform stream to decompress the file.
 | [options.check] | <code>Boolean</code> | <code>false</code> | enable write check |
```

Comment:
...and I dunno if it fits within the remit of this PR, but I wonder if the behaviour of `exports.mock` in `lib/validate.js` needs to be modified? (do any of the unit-tests exercise `options.check` being false?)